src/Moq/Moq.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```

Comment:
Next step should be to move to NuGetizer ;)

Reply:
Will look into it!